identity_data/src/commonMain/kotlin/net/thechance/mena/identity/data/utils/safeWrapper.kt

```diff
@@ -0,0 +1,23 @@
+package net.thechance.mena.identity.data.utils
+
+import io.ktor.client.plugins.ClientRequestException
+import io.ktor.http.HttpStatusCode
+import net.thechance.mena.identity.domain.exception.InvalidCredentialsException
+import net.thechance.mena.identity.domain.exception.UnAuthorizedException
+import net.thechance.mena.identity.domain.exception.UnknownException
+import net.thechance.mena.identity.domain.exception.UserIsBlockedException
+
+suspend fun <T> safeWrapper(function: suspend () -> T): T {
```

Comment:
I suggest changing the name of the argument `function`
```suggestion
suspend fun <T> safeWrapper(block: suspend () -> T): T {
```